en/about/faq.md

```diff
@@ -0,0 +1,43 @@
+# Frequently Asked Questions (FAQ)
+
+## Users
+
+<dl>
+  <dt>How efficient is MAVLink?</dt>
+  <dd>MAVLink is a very efficient protocol. It has just 8 bytes overhead per packet, including start sign and packet drop detection.</dd>
+
+  <dt>How many vehicles does MAVLink support?</dt>
+  <dd>255 vehicles, with system IDs ranging from 1 to 255 (0 is not a valid vehicle ID).
+    <br><b>Note:</b> Strictly speaking MAVLink supports 255 concurrent <em>systems</em>, and these can include a mix of vehicles, GCS, antenna trackers and other hardware.</dd>
+
+  <dt>Where can I use MAVLink?</dt>
+  <dd>MAVLink has been shown to work on multiple microcontrollers and operating systems, including ARM7, ATMega, dsPic, STM32 and Windows, Linux, MacOS, Android and iOS.</dd>
+
+  <dt>How safe is MAVLink?</dt>
+  <dd>Very. MAVLink has been used since 2009 to communicate between many different vehicles, ground stations (and other nodes) over varied and challenging communication channels (high latency/noise). It provides methods for detecting packet drops, and the well-established ITU X.25 checksum for packet corruption detection. MAVLink 2, further adds authentication using <a href="../guide/message_signing.md">message signing</a>.</dd>
+</dl>
+
+
+## Developers
+
+<dl>
+  <dt>Can I use MAVLink in a closed source application without copyright issues?</dt>
+  <dd>Yes, without any limitations. The LGPL license only requires you to contribute back changes to <em>MAVLink itself</em>, not any code that uses MAVLink. So you are not required to publish any of your application code.</dd>
+
+  <dt>How does MAVLink detect and decode messages in the byte stream?</dt>
+  <dd>MAVLink waits for the packet start sign, then reads the packet length and matches the checksum after n bytes. If the checksum matches, it returns the decoded packet and waits again for the start sign. If bytes are altered or lost, it will drop the current message and continue the next try on the following message.</dd>
+
+  <dt>MAVLink uses only one start sign - isn't this less safe than using two or three start signs?</dt>
+  <dd>No. A packet start sign is always used to "sync up" to the start of a message. To actually "prove" that the received data is a message and not random noise, the message length and check sum is needed. Using more start signs would allow to better “guess” a packet start in a random noise channel. Since MAV telemetry is however not random noise, using more start signs does not help to increase the certainty about a packet start, because if the decoding state machine does only depend on the packet start sign, it will reject all packets containing the start sign in their payload. 
```

Comment:
This explanation is hard to understand. In particular this sentence is confusing:

> Since MAV telemetry is however not random noise, using more start signs does not help to increase the certainty about a packet start, because if the decoding state machine does only depend on the packet start sign, it will reject all packets containing the start sign in their payload.

I'm not sure the logic follows: "we might detect start sign in message" -> "so we use CRC for rejection" -> "so we might as well have just one start sign and save the bytes." Yes you must use the CRC for rejection to avoid this systemic problem, but whether or not that is the case, you'll have a better chance of finding the start of the message with more start signs?

Even if I am wrong, the fact that I can doubt means that the explanation doesn't work.

Either way, we could perhaps position it as a tradeoff - ie define "safe" as meaning "being able to reliably reject incorrect messages" rather than "reliably detect start of message".

How about:

> No. We use the CRC check to reliably determine whether a complete message has been received. Using additional start signs may increase likelihood of detecting the start point, but would provide no greater certainty of message validity. Since extra signs would increase bytes on the communication link, we choose not to use them.

Reply:
@LorenzMeier Thanks for your other points, but what about this question?

Also https://github.com/mavlink/mavlink-devguide/pull/70#discussion_r200552463

Reply:
OK, so I decided to change this as indicated above: https://github.com/mavlink/mavlink-devguide/pull/70/commits/3dac326da5ee3e4dca5f6642c0fb17c75feca365

Then you can just review it fresh.